demos/k8s-demo/etc/secretless.yml

```diff
@@ -1,6 +1,6 @@
-version: 2
+version: "2"
```

Comment:
All of the ymls should use `version: 2` without quotes (it's not needed)

Reply:
why does it matter if we have the quotes or not? it works both ways